### PR TITLE
Change Phalanximine to be more complex, increase Arithrazine damage

### PR DESCRIPTION
**Pull request by SlamBamActionman**
## About the PR

This PR changes Phalanximine to be a more interesting medicine to use in treatment, by making it easy to use for small doses but being more demanding when healing higher amounts. 

- Metabolism rate is set to 0.1 (was 0.5). 
- Healing is reduced to 0.3 Cellular per tick (was 1)
- Damage is reduced to 0.15 Radiation per tick (was 1), 0.15 Caustic per tick
- Phalanx now has an OD threshold of 11u, after which it deals 0.2 Caustic per tick.
- Phalanx now has a negative reaction with Arithrazine, dealing 0.3 Caustic if both are in the system at the same time.
- Vomiting has been removed from Phalanx.

- Arithrazine has had its Brute damage increased from 0.5 to 1.5 per tick. 

- Sigynate now has a negative reaction with Arithrazine, with a 10% vomit chance per tick if both are in the system at the same time.

## Why / Balance

Phalanx has been a pretty tame drug to administer for a damage type that is one of the rarest in the game. Cellular is most commonly found in small amounts after a a slime attack, with some more build-up seen with eating brains. The PR #32136 intends to introduce a higher burst of Cellular that is most likely to be found in singular alive crew. 

The combined changes to Phalanx makes it easy to administer for small doses, but requires a longer treatment process for higher amounts of damage. 5u will heal 15 Cellular, so a doctor can administer 10u for a safe 30 Cellular heal. If a doctor has to heal higher amounts of Cellular the patient will be required to wait for the duration of the first injection before another can be made. Vomiting was removed due to the higher likelihood of it triggering over a longer duration, which makes it much harder to dose it and would require constant doctor supervision. 

The changes to Arithrazine is to make it less of a "inject a whole syringe no matter what" drug; a full syringe would deal 15 Brute damage, which since it's spread out over the damage category means it can be healed by a single brute pack (or just walked off via passive healing). This PR changes this to 45 damage for a full syringe, which should make it more appealing to pair with Bicardine or simply using more careful dosing. 

To prevent the post-Phalanx treatment to be a simple syringe of Arithrazine and Sigynate combined, there's now a vomit chance if the two are metabolised together. Considering the rarity of Caustic and Radiation it should be fairly uncommon to need to treat both at the same time, and even then the effectiveness of the drugs means you don't need a lot, making them be out of your system fast.

## Technical details

Yaml

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Phalanximine now has an overdose threshold and works slower, along with other minor tweaks.
- tweak: Arithrazine now deals increased damage during treatment.

### CODE REVIEW

**Comment by mirrorcult** [comment]
changes dont go far enough, i dont really think this will affect phalanx gameplay at all

**Comment by Aisu9** [comment]
> changes dont go far enough, i dont really think this will affect phalanx gameplay at all

paired with the new implant checking backlash, it will severely punish security for randomly check by making the genetic damage healing trice as long

**Comment by mirrorcult** [comment]
IMO it should probably both require cryo and be slower

**Comment by mirrorcult** [comment]
> > changes dont go far enough, i dont really think this will affect phalanx gameplay at all
> 
> paired with the new implant checking backlash, it will severely punish security for randomly check by making the genetic damage healing trice as long

yes, im obviously aware of those changes, im commenting on this PR specifically because of them, im saying making it only take a bit longer isnt enough of a deterrent

**Comment by SlamBamActionman** [comment]
> IMO it should probably both require cryo and be slower

I'm hesitant to set this as Cryo-only since that bars off Cellular healing behind both Chemistry knowledge (which at least has guidebook info on how to make, and isn't difficult to produce) and Cryo knowledge (which is a lot less intuitive). It would suck if you got a medbay of fairly inexperienced medical personnel and none of them know how to properly operate the Cryo chamber.

**Comment by thebadman4662** [comment]
> > IMO it should probably both require cryo and be slower
> 
> I'm hesitant to set this as Cryo-only since that bars off Cellular healing behind both Chemistry knowledge (which at least has guidebook info on how to make, and isn't difficult to produce) and Cryo knowledge (which is a lot less intuitive). It would suck if you got a medbay of fairly inexperienced medical personnel and none of them know how to properly operate the Cryo chamber.

It already is behind chemistry knowledge, all those fools out there walking past rad medkits with their 15u of Pha while being on 70 cellular damage after slimes. 🤣 

> It would suck if you got a medbay of fairly inexperienced medical personnel and none of them know how to properly operate the Cryo chamber.

Maybe they finally will learn if they will need to use it, playing the game is more or less behind engi properly operating energy generation and if they are all inexperienced you do not really get to play.

**Comment by mirrorcult** [comment]
> > IMO it should probably both require cryo and be slower
> 
> I'm hesitant to set this as Cryo-only since that bars off Cellular healing behind both Chemistry knowledge (which at least has guidebook info on how to make, and isn't difficult to produce) and Cryo knowledge (which is a lot less intuitive). It would suck if you got a medbay of fairly inexperienced medical personnel and none of them know how to properly operate the Cryo chamber.

cellular is supposed to be not easy or guaranteed to fix, gating it behind cryo seems completely reasonable

**Comment by BramvanZijp** [comment]
A cellular cryo chem already exists. Also, arith should not cause vomitting as hyronalin already does, with arith getting rid of that in exchange for dealing some brute damage.

**Comment by BramvanZijp** [comment]
> A cellular cryo chem already exists. Also, arith should not cause vomitting as hyronalin already does, with arith getting rid of that in exchange for dealing some brute damage.

Id say a better idea is to keep the vomit chance on phalanx, and have doxa, the cryo chem, be the better alternative to dealing with cellular in exchange for it being a cryo chem.

**Comment by thebadman4662** [comment]
> > A cellular cryo chem already exists. Also, arith should not cause vomitting as hyronalin already does, with arith getting rid of that in exchange for dealing some brute damage.
> 
> Id say a better idea is to keep the vomit chance on phalanx, and have doxa, the cryo chem, be the better alternative to dealing with cellular in exchange for it being a cryo chem.

So basically change nothing cause all goals are already achieved? :trollface:

**Comment by BramvanZijp** [comment]
On my PC now, so I'll just leave one big comment to explain why in my opinion, this PR should not be merged at all.
Its a PR that tries to make the burst genetic damage dealt by another PR that is not even merged yet, harder to heal.
Which would have been fine if this PR did not completely overlook every other balance aspect of the game except that specific case.
- This PR will completely remove consistant radiation chems from the game. The entire point of arithrazine is to be a better version of hyronalin, dealing slight brute damage in exchange for not being able to be randomly vomitted out. This PR seems to only think of arith as "step 2 to heal cellular damage after phalanx is used", with terms such as "moving the vomit chance", with complete disregard for the MANY situations in which radiation damage needs to be healed without the involvement of any cellular damage, which is now impossible to do without having a chance of the chemical being vomitted out.
- This PR seems to think that Phalanx is the only heal to cellular damage, it is not. Doxa is a cryo chem which is specifically designed to deal with cellular damage. As such it is in my opinion fine for phalanx to be inconsistant with a vomit chance, while doxa, the more advanced chemical follows the same path the rad meds do, the easier one is able to be vomitted out while the advanced one has a seperate criteria you need to deal with (Brute Damage / Cryo) I am also worried that if this does get merged and doxa becomes the new standard for cellular damage, another PR will be opened to then also nerf doxa, which should not happen purely because we want to punish security for random implant checking, at that point changes to that original PR should be made instead.
- This PR will make it much harder to deal with other sources of cellular damage, such as slimes. This, especially on lowpop will essentially force medical to have cryo up and running by the time the first slime wave happens, and as mentioned in the comments of this PR, cryo is not something people are too knowledgable on. I am genuinely concerned of the applications these changes will have on for example Salamander during low-pop hours, which I play, where sometimes there is just 1 or even no medical staff. Speaking of ingame applications that will happen as a side effect of this PR, nukies. Especially in warops, radiation is something commonly used against nukies with uranium spears, etc. To combat this, a nukie agent should be able to prepare chemicals to deal with this damage, but oh wait, theres a now a chance that regardless of them making hyro or arith, it will cause the radiation victim to vomit, requiring the injection of more anti-rad chems plus also making them lose food and water, which can cause major issues for nuclear operatives.

In general, I understand what this PR tries to do, but it does so in a way that completely ignores the very real balance impact these changes will have in several other key areas of the game. If you think the damage from the implanter PR will be too easy to heal, either increase that damage or add extra damage types that PR deals, such as caustic. But in my opinion, merging this PR will not be healthy for the game, and just prompt somebody else to make a revert PR, with the issues I mentioned persisting in-game untill that revert is merged.

**Comment by BramvanZijp** [comment]
> > > A cellular cryo chem already exists. Also, arith should not cause vomitting as hyronalin already does, with arith getting rid of that in exchange for dealing some brute damage.
> > 
> > 
> > Id say a better idea is to keep the vomit chance on phalanx, and have doxa, the cryo chem, be the better alternative to dealing with cellular in exchange for it being a cryo chem.
> 
> So basically change nothing cause all goals are already achieved? :trollface:

I do not think this PR should be merged, no. Read my above comment as to why.

**Comment by mirrorcult** [comment]
cellular from slimes should be removed, i think they were added after cellular was already easy to heal so it wasn't really thought of as an issue

the arith changes probably shouldnt be here tbh, im not exactly sure what effect they will have but it seems like too much of a shakeup when this is targeting phalanx mostly

with doxa existing which i forgot about (idk if ive ever seen it used) i think phalanx should just not heal cellular at all

**Comment by BramvanZijp** [comment]
> cellular from slimes should be removed, i think they were added after cellular was already easy to heal so it wasn't really thought of as an issue
> 
> the arith changes probably shouldnt be here tbh, im not exactly sure what effect they will have but it seems like too much of a shakeup when this is targeting phalanx mostly
> 
> with doxa existing which i forgot about (idk if ive ever seen it used) i think phalanx should just not heal cellular at all

The sole reason phalanx EXISTS is to heal cellular, just less effectively then doxa due to the vomitting, simular to hyro and arith as mentioned

And yes, the arith changes should not be here, but this PR is so solely focussed on phalanx that it only sees arith as the chem you use after you use phalanx, not as its own very common chem that will be drastically nerfed by this.

**Comment by SlamBamActionman** [comment]
> And yes, the arith changes should not be here, but this PR is so solely focussed on phalanx that it only sees arith as the chem you use after you use phalanx, not as its own very common chem that will be drastically nerfed by this.

I think it is a bit disingenuous to assume that I did not take normal radiation treatment into account; yes, Arith deals brute damage, but it's so tiny in comparison to the radiation effectiveness that you can just mix in a tiny bit of bicar or (more likely) let it self-heal. Puking would be an additional nerf, but other than the hunger effect all it does is remove a certain reagent amount, making it fairly trivial to just pump someone full with arith.

All that being said, the PR *was* made under the assumption that Cellular was an uncommon but not very rare damage type. Were it to be removed from yellow slimes I think I would be more comfortable to make much more sweeping changes to how Cellular is treated. So I'm going to explore in that direction since I agree with mirror that removing it from slimes would make it a more unique damage type.

**Comment by mirrorcult** [comment]
> 
> The sole reason phalanx EXISTS is to heal cellular,

this is not true, the sole reason phalanx EXISTS is because it was bulk ported from /vg/'s reagents, where phalanx was an anti-cancer chem. at some point years after it was added when it still had nothing (because the equivalent vg feature was never ported) someone randomly decided to bolt healing cellular onto it and now we're here.

**Comment by BramvanZijp** [comment]
> > And yes, the arith changes should not be here, but this PR is so solely focussed on phalanx that it only sees arith as the chem you use after you use phalanx, not as its own very common chem that will be drastically nerfed by this.
> 
> I think it is a bit disingenuous to assume that I did not take normal radiation treatment into account; yes, Arith deals brute damage, but it's so tiny in comparison to the radiation effectiveness that you can just mix in a tiny bit of bicar or (more likely) let it self-heal. Puking would be an additional nerf, but other than the hunger effect all it does is remove a certain reagent amount, making it fairly trivial to just pump someone full with arith.
> 
> All that being said, the PR _was_ made under the assumption that Cellular was an uncommon but not very rare damage type. Were it to be removed from yellow slimes I think I would be more comfortable to make much more sweeping changes to how Cellular is treated. So I'm going to explore in that direction since I agree with mirror that removing it from slimes would make it a more unique damage type.

If you think ariths downsides arent enough, then increase the damage it deals. We already have a puking antirad chem in hyronalin, making the other one also cause puking while also dealing damage is not something we should be doing IMO, and its ESPECIALLY not in the scope of a PR designed to nerf phalanx

**Comment by SlamBamActionman** [comment]
> If you think ariths downsides arent enough, then increase the damage it deals. We already have a puking antirad chem in hyronalin, making the other one also cause puking while also dealing damage is not something we should be doing IMO, and its ESPECIALLY not in the scope of a PR designed to nerf phalanx

I will be looking into other options for Phalanx/Arithrazine, but I feel I made a fairly relevant argument for why I made the change to Arithrazine. You may disagree with the change and that's fair, but taking Arithrazine into consideration with how it interacts with Phalanx is _definitely_ in scope.

**Comment by BramvanZijp** [comment]
> > If you think ariths downsides arent enough, then increase the damage it deals. We already have a puking antirad chem in hyronalin, making the other one also cause puking while also dealing damage is not something we should be doing IMO, and its ESPECIALLY not in the scope of a PR designed to nerf phalanx
> 
> I will be looking into other options for Phalanx/Arithrazine, but I feel I made a fairly relevant argument for why I made the change to Arithrazine. You may disagree with the change and that's fair, but taking Arithrazine into consideration with how it interacts with Phalanx is _definitely_ in scope.

You provided a fairly relevant argument for arithrazine in the context of it being used alongside phalanx, which is AT MOST 10% of its use cases. If you can provide an argument to change it that applies to the other 90%+ use cases I might change my stance on this, but even then I find it highly unlikely I will agree to making arithrazine WORSE then hyronalin, which is its weaker precursor (Even explained in the chem description.) Yes, you made arith WORSE then hyro. Hyronalin heals 1 rad damage per tick with a 2% chance to vomit each tick. Arith heals 3 rads per tick with a 10% chance to vomit, plus dealing 0.5 brute damage every tick. Arith multiplies the healing by 3 but the vomitting by 5 now (plus the brute damage), so EVEN IF vomitting by arith was a justified change (I disagree), this is FAR too much. If you disagree, please attach a video of you healing a character with 50 radiation damage (Not unlikely to occur) using arithrazine with the changes applied by this PR and then tell me the change you made is good for the game balance after cleaning up the massive puddle of vomit that will leave on the floor, especially if compared by healing it with hyronalin, since I expect you will use more arith then hyro based on the vomit chance.

And before you say "Ill just reduce the vomit chance", reduce it to 0. Arith is fine as is currently balance-wise and does not deserve nor need this change, especially in a PR whose primary scope is a different chemical.

**Comment by SlamBamActionman** [comment]
> > **I will be looking into other options for Phalanx/Arithrazine**, but I feel I made a fairly relevant argument for why I made the change to Arithrazine. You may disagree with the change and that's fair, but taking Arithrazine into consideration with how it interacts with Phalanx is _definitely_ in scope.
> 
> You provided a fairly relevant argument for arithrazine in the context of it being used alongside phalanx, which is AT MOST 10% of its use cases. If you can provide an argument to change it that applies to the other 90%+ use cases I might change my stance on this, but even then I find it highly unlikely I will agree to making arithrazine WORSE then hyronalin, which is its weaker precursor (Even explained in the chem description.) Yes, you made arith WORSE then hyro. Hyronalin heals 1 rad damage per tick with a 2% chance to vomit each tick. Arith heals 3 rads per tick with a 10% chance to vomit, plus dealing 0.5 brute damage every tick. Arith multiplies the healing by 3 but the vomitting by 5 now (plus the brute damage), so EVEN IF vomitting by arith was a justified change (I disagree), this is FAR too much. If you disagree, please attach a video of you healing a character with 50 radiation damage (Not unlikely to occur) using arithrazine with the changes applied by this PR and then tell me the change you made is good for the game balance after cleaning up the massive puddle of vomit that will leave on the floor, especially if compared by healing it with hyronalin, since I expect you will use more arith then hyro based on the vomit chance.
> 
> And before you say "Ill just reduce the vomit chance", reduce it to 0. Arith is fine as is currently balance-wise and does not deserve nor need this change, especially in a PR whose primary scope is a different chemical.

I already said I was going to change the way the chems work, so please, this level of intensity is not something I intend to engage with in discussion. Phalanx may be 10% of Arithrazine's usecase, but Arithrazine is 90% of Phalanx's usecase the way it is right now, so again, it's definitely worth considering how the two chemicals interact.

**Comment by thebadman4662** [comment]
Pha 10% use case of Ari? Other than field bottle/syringe treatment nobody uses it unless chemlab is mega unrobust, anyone sane either gets Dox or does not get treated at all cause they and doctors are clueless what is Pha or that radkits have syringes of them.

I bet average botany shift uses more Pha than double digit of medbay shifts combined, its use is simply that low unless there were many yellow slime infestations and again thats assuming no Dox, when slimes SCREAM for cryox and dox healing due to all the spread and types.

**Comment by BramvanZijp** [comment]
> > > **I will be looking into other options for Phalanx/Arithrazine**, but I feel I made a fairly relevant argument for why I made the change to Arithrazine. You may disagree with the change and that's fair, but taking Arithrazine into consideration with how it interacts with Phalanx is _definitely_ in scope.
> > 
> > 
> > You provided a fairly relevant argument for arithrazine in the context of it being used alongside phalanx, which is AT MOST 10% of its use cases. If you can provide an argument to change it that applies to the other 90%+ use cases I might change my stance on this, but even then I find it highly unlikely I will agree to making arithrazine WORSE then hyronalin, which is its weaker precursor (Even explained in the chem description.) Yes, you made arith WORSE then hyro. Hyronalin heals 1 rad damage per tick with a 2% chance to vomit each tick. Arith heals 3 rads per tick with a 10% chance to vomit, plus dealing 0.5 brute damage every tick. Arith multiplies the healing by 3 but the vomitting by 5 now (plus the brute damage), so EVEN IF vomitting by arith was a justified change (I disagree), this is FAR too much. If you disagree, please attach a video of you healing a character with 50 radiation damage (Not unlikely to occur) using arithrazine with the changes applied by this PR and then tell me the change you made is good for the game balance after cleaning up the massive puddle of vomit that will leave on the floor, especially if compared by healing it with hyronalin, since I expect you will use more arith then hyro based on the vomit chance.
> > And before you say "Ill just reduce the vomit chance", reduce it to 0. Arith is fine as is currently balance-wise and does not deserve nor need this change, especially in a PR whose primary scope is a different chemical.
> 
> I already said I was going to change the way the chems work, so please, this level of intensity is not something I intend to engage with in discussion. Phalanx may be 10% of Arithrazine's usecase, but Arithrazine is 90% of Phalanx's usecase the way it is right now, so again, it's definitely worth considering how the two chemicals interact.

What if the vomit chance on arith only stayed if phalanx is also in their system? That way you keep your desired effect without needlessly nerfing arith into the ground at the same time.

**Comment by BramvanZijp** [comment]
Alternatively, what if phalanx dealt caustic instead of rads. Sigy is rarer to see in a medbay so thats another option to consider

**Comment by SlamBamActionman** [comment]
The new values are the following:

Metabolism rate: 0.1 (= 5u takes 50 seconds)
Healing is set to: -0.3 Cellular per tick (= 5u heals 15 Cellular)
Damage is set to: 0.15 Radiation, 0.15 Caustic (= 5u deals 7.5 Radiation, 7.5 Caustic)
OD is set at 11; going over deals an additional 0.2 Caustic (= a dose of 15u deals a total of 22.5 Radiation, 30.5 Caustic)
If Arithrazine is in the system at the same time as Phalanximine, each tick deals an additional 0.3 Caustic.

Arithrazine now deals 1.5 Brute per tick, increased from 0.5. This increase is to give Arithrazine actual impact; previously a 15u dose of Arithrazine would heal 90 radiation, and have 15 brute (5 in each category) which meant you could be healed by a single brute pack or just have it heal on its own. Now 15u deals 45 brute, which still makes brute packs convenient, but Bicardine also becomes a viable alternative, or simply more careful dosing of Arithrazine.

If Sigynate and Arithrazine are in the system at the same time, there's a 10% chance per tick to vomit. Considering the rarity of both caustic and radiation, as well as the effective healing and fast metabolism of the medicine, having both in your system at the same time should be a rare occasion that probably really only happens with Phalanximine.

**Comment by thebadman4662** [comment]
Hmm, I kind of fail to see normal use for Pha now outside of botany chem. Its made so weak the treatment from radkits is reserved for most extreme niche survival situations. 

Ari treatments will require some Tri/Poly/Cryox treatment alongside I suppose, I do not feel neither for or against those changes. I suppose it was weird how that 0.5 brute damage was basically so small you could not care about it entirely in 99%+ cases so I guess I am actually bit for it.

In other words Dox keeps on winning. :godmode:

**Comment by SlamBamActionman** [comment]
> Hmm, I kind of fail to see normal use for Pha now outside of botany chem. Its made so weak the treatment from radkits is reserved for most extreme niche survival situations.
> 
> Ari treatments will require some Tri/Poly/Cryox treatment alongside I suppose, I do not feel neither for or against those changes. I suppose it was weird how that 0.5 brute damage was basically so small you could not care about it entirely in 99%+ cases so I guess I am actually bit for it.
> 
> In other words Dox keeps on winning. :godmode:

Cryo is going to beat out Phalanximine every day of the week, but is of course gated by knowledge and space. I think that's an okay tradeoff. In relation to #32136 it means you can't just give Security a jug and have them be happy. 

The change in metabolism is just that; it acts slower, but the potency per unit of reagent has actually increased. Previously 5u healed 10 Cellular, now it heals 15, just over a longer duration.

**Comment by thebadman4662** [comment]
> > Hmm, I kind of fail to see normal use for Pha now outside of botany chem. Its made so weak the treatment from radkits is reserved for most extreme niche survival situations.
> > Ari treatments will require some Tri/Poly/Cryox treatment alongside I suppose, I do not feel neither for or against those changes. I suppose it was weird how that 0.5 brute damage was basically so small you could not care about it entirely in 99%+ cases so I guess I am actually bit for it.
> > In other words Dox keeps on winning. :godmode:
> 
> Cryo is going to beat out Phalanximine every day of the week, but is of course gated by knowledge and space. I think that's an okay tradeoff. In relation to #32136 it means you can't just give Security a jug and have them be happy.

Technically you reinforced that, but it assumes chemist have braincells to give sec Ari-Sig soaked cigs/smokes. :trollface: 
But in practice we all know average IQ of spessman, let alone chemist.

**Comment by thebadman4662** [comment]
I thought the other day, why not simply make Pha purge Ari and Hyro from bloodstream?

Only options other than proper treatment would be
- Using Ult(which is so fucking bad and thus not used outside ghetto chem)
- Cryopods and 50/50 mix, but then why won't you use Dox?
- Smokes. Its slow as fuck with 0.02-0.03u healing AT BEST per second.

**Comment by Pumkin69** [comment]
i like the direction but i think you should also make it a cryo only chemical. Cryo isn't used enough and making it so sec cant just get jugs of arith and phalanx and instead having to go to med for cryo will absolutely kill random implant checking with the accommodating pr.

**Comment by BramvanZijp** [comment]
> i like the direction but i think you should also make it a cryo only chemical. Cryo isn't used enough and making it so sec cant just get jugs of arith and phalanx and instead having to go to med for cryo will absolutely kill random implant checking with the accommodating pr.

There already IS a dedicated cryochem for cellular damage.

**Comment by Pumkin69** [comment]
> > i like the direction but i think you should also make it a cryo only chemical. Cryo isn't used enough and making it so sec cant just get jugs of arith and phalanx and instead having to go to med for cryo will absolutely kill random implant checking with the accommodating pr.
> 
> There already IS a dedicated cryochem for cellular damage.

wow i didnt even know that. On further consideration i think its fine as is but would like to see hyro also get the negative reaction when mixed with phalanx.

**Comment by BramvanZijp** [comment]
> > > i like the direction but i think you should also make it a cryo only chemical. Cryo isn't used enough and making it so sec cant just get jugs of arith and phalanx and instead having to go to med for cryo will absolutely kill random implant checking with the accommodating pr.
> > 
> > 
> > There already IS a dedicated cryochem for cellular damage.
> 
> wow i didnt even know that. On further consideration i think its fine as is but would like to see hyro also get the negative reaction when mixed with phalanx.

The negative reaction is that hyro causes vomitting, which will then flush out the phalanx which will be very noticable as you apply it in small doses and it motabilizes slowly.

**Review by VasilisThePikachu — APPROVED** [review]
